src/main/java/seedu/duke/exception/DuplicateClientException.java

```diff
@@ -0,0 +1,14 @@
+package seedu.duke.exception;
+
+import static seedu.duke.Messages.MESSAGE_DUPLICATE_CLIENT;
+
+//@@author OVReader
```

Comment:
I think @@author is supported in '/*' format.
Can consider adding inside the header.